analysis_scripts/analysis_utils.R

```diff
@@ -101,6 +101,65 @@ read_gender_files <- function(in_file){
 }
 
 
+#' Compute first v last bootstrap CI
+#' this works by taking a random subset of articles per year
+#' and calculating the bootstrap mean, upperCI and lowerCI
+#' its assumed that there exists a column called author_pos
+#' where there are only two values -- first / last
```

Comment:
These seem to be important conditions for the `author_pos`. I think it would be good to add a `stopifnot` check that there are only two values in this column and they are these two. Maybe I'm wrong, but sounds like you can easily make a mistake by providing a wrong set of values here.

Reply:
good point! added here: https://github.com/greenelab/nature_news_disparities/pull/23/commits/476dcc9d697721a2faf88ee369e8f9ccedf01d80